AxonDeepSeg/download_model.py

```diff
@@ -4,24 +4,27 @@
 
 
 def download_model(destination = None):
+    sem_release_version = 'r20211111v2'
+    tem_release_version = 'r20211111v3'
+    om_release_version = 'r20211209'
+    
     if destination is None:
-        sem_destination = Path("AxonDeepSeg/models/default_SEM_model")
-        tem_destination = Path("AxonDeepSeg/models/default_TEM_model")
-        model_seg_pns_bf_destination = Path("AxonDeepSeg/models/model_seg_pns_bf")
+        sem_destination = Path("AxonDeepSeg/models/model_seg_rat_axon-myelin_sem")
+        tem_destination = Path("AxonDeepSeg/models/model_seg_mouse_axon-myelin_tem")
+        om_destination = Path("AxonDeepSeg/models/model_seg_rat_axon-myelin_bf")
     else:
-        destination = convert_path(destination)
-        sem_destination = destination / "default_SEM_model"
-        tem_destination = destination / "default_TEM_model"
-        model_seg_pns_bf_destination = destination / "model_seg_pns_bf"
+        sem_destination = destination / "model_seg_rat_axon-myelin_sem"
+        tem_destination = destination / "model_seg_mouse_axon-myelin_tem"
+        om_destination = destination / "model_seg_rat_axon-myelin_bf"
```

Comment:
I noticed that too today! Your call, I had no idea what bf meant. Though, it's mostly a backend nomenclature so it's not *super* important I think, your call!

Reply:
I think I would prefer the BIDS nomenclature. We can decide in meeting with everyone tomorrow.
For the user it would appear at 2 places:
- "OM" in the -m argument of the CLI
- The name of the repo for the model

Reply:
Done!